orchagent/neighorch.cpp

```diff
@@ -1203,6 +1249,19 @@ void NeighOrch::voqSyncAddNeigh(string &alias, IpAddress &ip_address, const MacA
     sai_attribute_t attr;
     sai_status_t status;
 
+    Port inbandPort;
+    if (!gPortsOrch->getInbandPort(inbandPort))
+    {
+        SWSS_LOG_ERROR("Failed to get inband port/VLAN");
+        return;
+    }
+
+    if (inbandPort.m_type == Port::VLAN && inbandPort.m_alias == alias)
+    {
+        SWSS_LOG_NOTICE("Skip inband VLAN neigh: %s", ip_address.to_string().c_str());
+        return;
+    }
+
```

Comment:
It will be great if comments are added for this block of code saying that "inband neighbors are not synced to CHASSIS_APP_DB for inband interface type VLAN since the inband neigibors are learned via arp learning within inband VLAN"

Reply:
Added the comment as suggested.